columnflow/inference/cms/datacard.py

```diff
@@ -315,7 +315,6 @@ def write(
     def write_shapes(
         self,
         shapes_path: str,
-        fill_empty_bins: float = 1e-5,
```

Comment:
Same comment as before, would be nice if we could update the doc string already now

Reply:
And same comment as above :) Having this in a dedicated PR could be cleaner.